### ISSUE TITLE
log-fe error retrieving logs

### ISSUE THREAD

**Issue by hgardneriv** [gliang/log-collector#3]
The UI has error retrieving logs, possibly related to [this](https://github.com/gliang/log-collector/issues/2) issue.

<img width="1735" alt="Screen Shot 2022-04-19 at 7 05 37 AM" src="https://user-images.githubusercontent.com/9706538/163999527-0edf628a-a1cd-43ad-ba4f-80aa34b271b8.png">

**Nice to have**: it would be helpful to provide default values in the UIs Server URLs field since the instructions specify which URLs to use. 

Suggestions on how to work around this?

**Comment by gliang** [comment]
Good catch. Please add a **.env.local** file in the project root directory with this line
`VITE_BACKEND_URL = "http://localhost:5001"`
